### PR TITLE
[Hot-fix] Update if statement for check EvalHook

### PR DESCRIPTION
**Pull request by harimkang**
### This PR includes :
- Update if statement for checking EvalHook

### CODE REVIEW

File: mpa/modules/hooks/adaptive_training_hooks.py [hunk 1]
```diff
@@ -55,10 +55,10 @@ def before_train_epoch(self, runner):
             iter_per_epoch = len(runner.data_loader)
             adaptive_interval = self.get_adaptive_interval(iter_per_epoch)
             for hook in runner.hooks:
-                if isinstance(hook, EvalHook):
+                if isinstance(hook, EvalHook) or 'EvalHook' in str(hook):
                     hook.interval = adaptive_interval
                     logger.info(f"Update Validation Interval: {adaptive_interval}")
-                elif isinstance(hook, LrUpdaterHook):
+                if isinstance(hook, LrUpdaterHook):
```
Comment (goodsong81) [inline mpa/modules/hooks/adaptive_training_hooks.py]: Why elif -> if?

Comment (harimkang) [reply mpa/modules/hooks/adaptive_training_hooks.py]: It was a mistake. I corrected it.

**Review by jaegukhyun — APPROVED** [review]
LGTM

File: mpa/modules/hooks/adaptive_training_hooks.py [hunk 2]
```diff
@@ -55,7 +54,7 @@ def before_train_epoch(self, runner):
             iter_per_epoch = len(runner.data_loader)
             adaptive_interval = self.get_adaptive_interval(iter_per_epoch)
             for hook in runner.hooks:
-                if isinstance(hook, EvalHook):
+                if 'EvalHook' in str(hook):
```
Comment (goodsong81) [inline mpa/modules/hooks/adaptive_training_hooks.py:57]: Could you check if this works?
`issubclass(type(hook), EvalHook)`

**Review by goodsong81 — APPROVED** [review]
LGTM, but there is alternatives.

**Pull request merged**